qa/rpc-tests/schnorr-activation.py

```diff
@@ -1,475 +0,0 @@
-#!/usr/bin/env python3
-# Copyright (c) 2015-2016 The Bitcoin Core developers
-# Copyright (c) 2017-2019 The Bitcoin developers
-# Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
looks like you removed rather than added the activation test?

Reply:
activation test is in schnorrsig.py rather than this file which would be very time consuming to port.